pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -44,21 +45,30 @@ func TestParseLocation(t *testing.T) {
 }
 
 func TestEmptyConfig(t *testing.T) {
-	registries, err := GetRegistries(&types.SystemContext{SystemRegistriesConfPath: "testdata/empty.conf"})
+	registries, err := GetRegistries(&types.SystemContext{
+		SystemRegistriesConfPath:    "testdata/empty.conf",
+		SystemRegistriesConfDirPath: "invalid",
```

Comment:
(Non-blocking: `testdata/this-does-not-exist` if that’s the kind of “invalid” this is? Also in the other cases.)